boards/adi/max32655fthr/max32655fthr_max32655_m4.dts

```diff
@@ -144,6 +144,12 @@
 	status = "okay";
 };
 
+&i2c1 {
+	status = "okay";
+	pinctrl-0 = <&i2c1_scl_p0_16 &i2c1_sda_p0_17>;
```

Comment:
Do these pins need to operate at 1.8 or 3.3V? If 3.3V, then you need to set `MAX32_VSEL_VDDIOH`